app/V1Module/presenters/ExercisesConfigPresenter.php

```diff
@@ -377,7 +377,8 @@ public function checkGetHardwareGroupLimits(string $id, string $runtimeEnvironme
   }
 
   /**
-   * Get a description of resource limits for an exercise for given hwgroup
+   * Get a description of resource limits for an exercise for given hwgroup.
+   * @DEPRECATED
```

Comment:
In the future (and possibly near one), endpoints marked as `DEPRECATED` should return `deprecated` flag in response envelope and this flag should be propagated also to our API Swagger documentation.